ts/boot/configureInstabug.ts

```diff
@@ -100,6 +100,28 @@ export const setInstabugProfileAttributes = (
   );
 };
 
-export const instabugLog = (log: string, typeLog: TypeLogs) => {
-  InstabugLogger[typeLog](log);
+// The maximum log length accepted by Instabug
+const maxInstabugLogLength = 4096;
+// margin used for numerate the chunks
+const numberMargin = 15;
+/**
+ * This method allows to log a string in the Instabug report. If the log is too long,
+ * the string will be splitted in chunks
+ * @param log the text that will be logged on istabug
+ * @param typeLog the type of the log
+ * @param tag a tag that can be used to identify the log
+ */
+export const instabugLog = (log: string, typeLog: TypeLogs, tag?: string) => {
+  const chunckSize =
+    maxInstabugLogLength - (tag ? tag.length : 0) - numberMargin;
+
+  const chunks = log.match(
```

Comment:
```suggestion
  const chunks = text.match(new RegExp("/(.|[\r\n]){1," + cs.toString() + "}", "g"));
```
This Regex doesn't take care of new lines or carriage returns (it could happen when we do `JSON.stringify(data, null, 4)` to have a pretty format)
I did some tests and with a string of 6327 of length, it chunks in 145 pieces instead of 2

Reply:
Can I have the example file?

Reply:
Changed in 8909a96